test/e2e/fishapp/dynamic_stack.go

```diff
@@ -43,20 +43,21 @@ const (
 	AppContainerPort                       = 9080
 	HttpProxyContainerPort                 = 8899
 	defaultAppImage                        = "public.ecr.aws/e6v3k1j4/colorteller:v1"
-	defaultHTTPProxyImage                  = "abhinavsingh/proxy.py:latest"
-	caCertScript                           = "certs/ca_certs.sh"
-	nodeCertScript                         = "certs/node_certs.sh"
-	genericNodeCertCfgFile                 = "certs/node_cert.cfg"
-	certsBasePath                          = "certs/"
-	certsCfgFileSuffix                     = "_cert.cfg"
-	certChainSuffix                        = "_cert_chain.pem"
-	certKeySuffix                          = "_key.pem"
-	caCertFile                             = "ca_cert.pem"
-	envoyCACertPath                        = "/certs/ca_cert.pem"
-	certCleanupScript                      = "certs/cleanup.sh"
-	sdsDeployScript                        = "certs/sds_provider.sh"
-	registerAgentIdentity                  = "certs/register_agent_entry.sh"
-	registerWorkloadIdentity               = "certs/register_workload_entry.sh"
+	// Change defaultHTTPProxyImage to ghcr.io/abhinavsingh/proxy.py:v2.4.0b3.dev31.ga062f80-linux.arm64.v8 for testing on ARM64
```

Comment:
While this is alright, I feel it is better we instead have a README under `test` folder with all the caveats and basic instructions on how to run the test suite similar to our VPC CNI repo. Unless someone looks in to the code and reads these comments, they will miss this. Instructions on how to run the integration tests is a common request for us and maybe we can use this opportunity to add them. Can be a separate PR as well...

Reply:
Sure, will move this to a section under Readme under test.